auditwheel/policy/__init__.py

```diff
@@ -27,12 +28,10 @@
     'manylinux2010_i686': ['linux_i686'],
 }
 
-# XXX: this could be weakened. The show command _could_ run on OS X or
-# Windows probably, but there's not much reason to inspect foreign package
-# that won't run on the platform.
-# if platform != 'linux':
-#     logger.critical('Error: This tool only supports Linux')
-#     sys.exit(1)
+if platform != 'linux':
+    printp('Running on {}. Note that this tool only '
```

Comment:
I guess wording needs some adjustment than...
" Note that this tool was designed to work mainly on Linux systems, So results on Non-Linux platforms might be incorrect"

Reply:
Do you mean that we should say that `auditwheel` can be used on non-Linux systems? Apologies if I misunderstood your comment but it would be helpful if you could elaborate on why you think this change in wording is necessary.

Reply:
No NO, just to make the sentence more softer... :D

Reply:
you can adjust as your wish

Reply:
If you want I can get this in as is

Reply:
I would love to hear your responses on the queries of Thomas's